server/src/main/java/com/openelements/benchscape/server/store/data/EnvironmentQuery.java

```diff
@@ -39,6 +39,9 @@ public record EnvironmentQuery(@Nullable String name,
                                @Nullable String osVersion,
                                @Nullable String jvmVersion,
                                @Nullable String jvmName,
+                               @Nullable OperationSystem osFamily,
+                               @Nullable String systemMemoryReadable,
+                               @Nullable Integer cores,
```

Comment:
`Integer systemProcessors` is what should be used for cores. Do not create a new variable (but It will make sense to create an issue for using the name everywhere)